corehq/apps/cloudcare/templates/formplayer/query_view.html

```diff
@@ -34,15 +34,25 @@
   </td>
   </td>
   <td class="col-sm-2">
-    <% if (input == "select1") { %>
-      <select class="query-field form-control"  data-receive="<%- receive %>">
-        <option value=""></option>
-        <% for (var i = 0; i < itemsetChoices.length; i++) { %>
-          <option value="<%- i %>" <% if (value === i) { %>selected<% } %>>
-            <%- itemsetChoices[i] %>
-          </option>
-        <% } %>
-      <select>
+    <% if (input == "select1" || input == "select") { %>
```

Comment:
does this need to be nested like this? I don't see any common pieces between `select1` and `select`

Reply:
True, will remove.